hack/istio/playbooks/README.adoc

```diff
@@ -48,13 +38,22 @@ You must choose between Ingress Route or Openshift Route there is a traffic gene
 - If you want to use ingress route (all traffic is going to hit ingress-gateway), you may use the following command:
 
 ```
-ansible-playbook bookinfo.yml -e istio_version=1.0.0 -e ingress_route=true -e traffic_generator=true -e rate=1 -vv
+ansible-playbook bookinfo.yml -e ingress_route=true -e traffic_generator=true -e rate=1 -vv
 ```
 
 === Kiali Traffic Generator with Openshift Route
 
 - If you want to use openshift route (kiali will report as "unknown"), you may use the following command:
 
 ```
-ansible-playbook bookinfo.yml -e istio_version=1.0.0 -e ingress_route=false -e traffic_generator=true -e rate=1 -vv
+ansible-playbook bookinfo.yml -e ingress_route=false -e traffic_generator=true -e rate=1 -vv
+```
+
+
+=== Kiali Traffic Generator with MongoDB
+
+- If you want to enable MongoDB for ratings, you may use the following command
+
+```
+ansible-playbook bookinfo.yml -e ingress_route=false -e traffic_generator=true -e rate=1 -e -vv
```

Comment:
What's the difference between this command and the one on line 49? Is there supposed to be an option to turn on mongodb that is missing here?